frontend/src/components/pages/GlobalCodeReviewDashboard/BaseQueryEntry.jsx

```diff
@@ -81,12 +83,8 @@ export default function BaseReviewQueueEntry({
           <ReviewersTable
             reviewerUsers={project.reviewerUsers}
             reviewerUserEmails={project.reviewerUserEmails}
-            usersThatReviewedSinceLastReviewRequest={
-              project.usersThatReviewedSinceLastReviewRequest
-            }
-            usersThatReviewedSinceLastReviewRequestEmails={
-              project.usersThatReviewedSinceLastReviewRequestEmails
-            }
+            usersThatReviewedSinceLastReviewRequest={reviewers}
```

Comment:
- Should the values passed to this not contain id's and not emails? I'm saying this looking at the first two props for ReviewersTable

- Would passing in both the reviewer ids and emails as props not make it easier to deal with? maybe you might not need the complex `filterUsers` function and more manipulation in the `ReviewersTable.jsx`?

Reply:
`ReviewersTable.jsx` is currently only catered for the competence reviews, and the dataset has two different reviewers, the PR reviews and the competence review, so I needed a way to allow the table to take in two different reviewers